torchci/lib/drciUtils.ts

```diff
@@ -135,7 +135,7 @@ export function getActiveSEVs(issues: IssueData[]): [IssueData[], IssueData[]] {
     (issue: IssueData) => issue.state === "open"
   );
   return _.partition(activeSEVs, (issue: IssueData) =>
-    issue.body.toLowerCase().includes("merge blocking")
+    issue.labels.includes("merge blocking")
```

Comment:
This probably conflicts with https://github.com/pytorch/test-infra/pull/5914, but using a label looks cleaner